README.md

```diff
@@ -1,3 +1,6 @@
 # parcel-examples
 
 🛠 Parcel examples using various frameworks
+
+## [Vue.js](https://vuejs.org)
+- [Parcel Basic Example with Vue.js](https://github.com/parcel-bundler/examples/tree/master/vue)
```

Comment:
Since `Vue` is the sub-directory of `parcel-bundler/examples` repo you could just do this `/Vue`, instead of putting the entire URL.

Reply:
- [x] Done in 91199d7